src/main/resources/views/CommercialVehicles/fail.hbs

```diff
@@ -167,7 +167,7 @@
                 <!-- /heading-info -->
             </div>
             <div class="header__right">
-                <img src="{{root}}/assets/images/dvsa_crest.png" alt="DVSA Crown" class="header__logo" id="logo"/>
+                <img src="{{root}}/assets/images/dvsa-crest-welsh-v2.png" alt="DVSA Crown" class="header__logo" id="logo"/>
```

Comment:
This logo was correct, as this is VTP30, it's failWelsh (VTP30W) that needs to have the crest changed.